client.js

```diff
@@ -78,25 +78,35 @@ function connect(EventSource) {
 
 }
 
-var strip = require('strip-ansi');
-
-var overlay;
-if (typeof document !== 'undefined' && options.overlay) {
-  overlay = require('./client-overlay');
+var reporter;
+// the reporter needs to be a singleton on the page.
```

Comment:
Could you expand this comment to say why?

Something that mentions multiple bundles will do it


LGTM aside from that